web/themes/new_weather_theme/assets/sass/components/radar.scss

```diff
@@ -59,10 +57,22 @@
   }
 }
 
+.wx-radar-container .cmi-map {
+  aspect-ratio: 3/2 !important;
+  height: auto !important;
+  width: auto !important;
+}
+  
+.wx-radar-container.wx-radar-container__expanded .cmi-map {
+  aspect-ratio: 3/4 !important;
+  height: auto !important;
+  width: auto !important;
+}
+
 @include at-media("tablet") {
   .wx-radar-container {
     &.wx-radar-container__expanded {
-      aspect-ratio: 1/1;
+      aspect-ratio: 1/1 !important;
     }
   }
 }
```

Comment:
As we talked about, the controls will go back to being absolutely conditioned so we don't have the cyan gap. We're adjusting the aspect ratios to include the controls in order to get it working correctly. I've adjusted the sizes slightly so that the ratios use integer numbers:

Breakpoint | Collapsed | Expanded 
-|-|-
Mobile| 1/1 |  3/5
Tablet | 3/4 | 3/4
Desktop | 1/1 | 3/2